impl/maven-impl/src/main/java/org/apache/maven/impl/model/DefaultModelInterpolator.java

```diff
@@ -129,23 +125,9 @@ protected List<String> getProjectPrefixes(ModelBuilderRequest request) {
                 : PROJECT_PREFIXES_3_1;
     }
 
-    String callback(
-            Model model,
-            Path projectDir,
-            ModelBuilderRequest request,
-            ModelProblemCollector problems,
```

Comment:
Parameter 'problems' is never used.

<img width="1872" alt="image" src="https://github.com/user-attachments/assets/4a616b2b-ca3b-4770-90af-084286d84b80" />